packages/ui-core/common/src/lib/http-params/build-http-params.ts

```
@@ -0,0 +1,48 @@
+		const paramKey = prefix ? `${prefix}[${key}]` : key;
+
+		if (value === null || value === undefined) {
+			// Skip null or undefined values
+			return;
+		}
+
+		if (Array.isArray(value)) {
+			// Append each array item individually
+			value.forEach((item) => {
+				if (typeof item === 'object') {
+					params = addToParams(params, item, paramKey);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)